### PR TITLE
Smaller aesthetic fixes to InternalTestCluster

### PR DESCRIPTION
**Pull request by ywelsch**
Smaller stuff that I noticed while working on other stuff...

### CODE REVIEW

File: test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java [hunk 1]
```diff
@@ -1649,10 +1638,7 @@ public synchronized String startNode(Settings.Builder settings) {
      * Starts a node with the given settings and returns it's name.
```
Comment (DaveCTurner) [inline test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java]: *its (it's wrong in some other places too)

**Review by DaveCTurner — COMMENTED** [review]
Changes LGTM, but tests are failing. Not sure if related or not.

**Comment by ywelsch** [comment]
The test failures were actually caused by this PR. Due to the now possibly more rapid expansion of clusters, we cannot always update the min_master_node setting before starting the new master-eligible nodes.

Assume for example that you have a one node cluster and you add two more nodes. We can then not set minimum_master_nodes to 2 on that single-node cluster, as we have a settings validation check that ensures that minimum_master_nodes cannot be set to a value higher than the number of master-eligible nodes in the cluster.

**Review by DaveCTurner — APPROVED** [review]
Makes sense, LGTM.

**Pull request merged**